apps/site/src/views/Home/sections/Intro/Intro.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import pin from "@/assets/images/index-card-pin.svg";
+import hackDoodle from "@/assets/images/hack-doodle.png";
+import styles from "./Intro.module.scss";
+
+export default function Intro() {
+	return (
+		<Container as="section">
+			<Row className={styles.intro + " mx-2"}>
```

Comment:
The container layout should handle the horizontal padding, so I think this doesn't need to be a row with additional margins, could just be a div as I previously suggested.